komga-webui/src/components/ThumbnailExplorerDialog.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <v-dialog v-model="input" scrollable>
+    <v-card :max-height="$vuetify.breakpoint.height * .9" dark>
+      <v-card-text>
+        <v-container fluid>
+          <v-row class="mb-2 d-flex flex-sm-row flex-column align-center justify-space-around">
+
+            <v-pagination
+              v-model="page"
+              :total-visible="perPage"
+              :length="Math.ceil(thumbnails.length/perPage)"
+            ></v-pagination>
```

Comment:
```suggestion
<v-card-title>
        <v-pagination
          v-model="page"
          :total-visible="perPage"
          :length="Math.ceil(thumbnails.length/perPage)"
        ></v-pagination>
      </v-card-title>
      <v-card-text>
        <v-container fluid>
          <v-row class="mb-2 align-center justify-space-around">
```

The `scrollable` prop on `v-dialog` works inside a `v-card-text`. Moving the pagination to `v-card-title` make it always showing, even when scrolling on `xs` screens.

Also removed the `flex-column`, since there are no `v-col`, the `v-row` will automatically move overflowing items on new rows. This way we get 2 images per row on mobile instead of 1.